lib/Doctrine/ORM/Configuration.php

```diff
@@ -561,7 +568,7 @@ public function setCustomDatetimeFunctions(array $functions)
     /**
      * Sets the custom hydrator modes in one pass.
      *
-     * @param array $modes An array of ($modeName => $hydrator).
+     * @param array<string, class-string> $modes An array of ($modeName => $hydrator).
```

Comment:
```suggestion
     * @param array<\Doctrine\ORM\AbstractQuery::HYDRATE_*, class-string> $modes An array of ($modeName => $hydrator).
```

It's mostly `\Doctrine\ORM\AbstractQuery::HYDRATE_*` constants, BUT yes, you can set custom once. So I don't know if I'm fully right